superset-frontend/src/components/Chart/DrillBy/DrillByMenuItems.tsx

```diff
@@ -138,10 +138,9 @@ export const DrillByMenuItems = ({
 
   useEffect(() => {
     if (open) {
-      ref.current?.input.focus({ preventScroll: true });
+      ref.current?.input?.focus({ preventScroll: true });
     } else {
       // Reset search input when menu is closed
-      ref.current?.setValue('');
```

Comment:
We don't need this anymore? It seems unrelated to the upgrade

Reply:
There is no setValue method for Input after the migration so this would give type errors.